packages/office-ui-fabric-react-experiments/src/components/Link/Link.scss

```diff
@@ -0,0 +1,44 @@
+@import '~office-ui-fabric-react/dist/sass/References';
+
+// Element: ms-Link root component
+.root {
```

Comment:
Yeah, I only put the link in because it was super simple and I wanted to make sure things were working. I expect it'll be replaced with something more informative